src/clockcache.c

```diff
@@ -1822,20 +1822,25 @@ clockcache_init(clockcache        *cc,   // OUT
    cc->heap_id = hid;
 
    /* lookup maps addrs to entries, entry contains the entries themselves */
-   cc->lookup =
-      TYPED_ARRAY_MALLOC(cc->heap_id, cc->lookup, allocator_page_capacity);
+   platform_memfrag memfrag_cc_lookup;
+   cc->lookup = TYPED_ARRAY_MALLOC_MF(
+      cc->heap_id, cc->lookup, allocator_page_capacity, &memfrag_cc_lookup);
    if (!cc->lookup) {
       goto alloc_error;
    }
+   cc->lookup_size = memfrag_size(&memfrag_cc_lookup);
```

Comment:
Here's the first instance of an init / deinit pair of calls, which now need to communicate the size of memory fragment allocated by init().
Like it's done on this line, few common structures now gain a new `size` field to track the memory fragment's size.